src/utils/context.ts

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export interface IBoardContext {
+  title: string;
+  description: string;
+  bgImage: string;
+}
```

Comment:
Interface docs? Does TS do that?

Reply:
I'll write a one-line description, it feels pretty self-explanation.